MiniTwitch.PubSub/PubSubClient.cs

```diff
@@ -469,9 +469,7 @@ private Task OnWsDisconnect()
     {
         Log(LogLevel.Warning, "Disconnected");
         _pingerToken.Cancel();
-        _pinger.Dispose();
         Log(LogLevel.Trace, "Pinger task disposed");
```

Comment:
Move this to .DisposeAsync()

Reply:
Whoops, I hallucinated that it was there. Will do!

Reply:
Nope, it is already there.

Reply:
I'm talking about the log line, it isn't there. Either move it or remove it from here since it doesn't make sense anymore

Reply:
Oh sorry, forgot that GH doesn't just show the line you're talking about. Will move

Reply:
Done in 728487082c46b330c955c810982478839fbc8ea9